### ISSUE TITLE
[BubbleChart] Cannot use keyboard to focus on bubbles in bubble chart

### ISSUE THREAD

**Issue by jaimiegorman** [carbon-design-system/carbon#7669]
## Environment

Test Tool Type: Manual test (AVT2)

## Detailed description

> What version of the Carbon Design System are you using?

> What did you expect to happen? Expect to be able to use keyboard to navigate across bubble chart, 

> What happened instead? Cannot use keyboard to navigate across bubble chart

> What WCAG 2.1 checkpoint does the issue violate? IBM CI 162 Checkpoint Violation: 2.1.1 Keyboard, IBM Guidance: https://www.ibm.com/able/guidelines/ci162/keyboard.html

## Steps to reproduce the issue

1. Login to RTWA employer-analytics page: https://g11ntest.wh-rtw.dev.watson-health.ibm.com/analytics/
2. Access RTWA employer homepage
3. Use keyboard (Tab key) to navigate in Exploring and comparing workplaces tab

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

## Additional information

- Screenshots or code
- 
![image](https://user-images.githubusercontent.com/78103524/106036881-e4d48200-60a3-11eb-928a-242c39db01c5.png)

- Notes
